include/swift/AST/RawComment.h

```diff
@@ -79,16 +79,17 @@ struct CommentInfo {
 };
 
 struct LineColumn {
-  uint32_t Line;
-  uint32_t Column;
+  uint32_t Line = 0;
+  uint32_t Column = 0;
+  bool isValid() const { return Line && Column; }
```

Comment:
Would it make sense to assert somewhere that we never create a `LineColumn` with a zero `Line` but a nonzero `Column`, or vice versa? I worry about one of these fields ending up with garbage in it without us noticing because the other is zeroed, and then something reading that garbage and doing the wrong thing.

Reply:
hmm, not sure it worths it. We plan to use `LineColumn` here as an internal implementation detail just like `CommentInfo` defined above so users won't directly interact with this type. I'll open future PRs to hook `LineColumn` up with AST APIs like `getLoc()`, etc.